src/clj/game/cards/hardware.clj

```diff
@@ -784,15 +784,17 @@
                                (when (= (get-in (get-card state card) [:counter :power]) 3)
                                  (system-msg state :runner "trashes Respirocytes as it reached 3 power counters")
                                  (trash state side card {:unpreventable true})))}]
-   {:effect (req (add-watch state :respirocytes
+   {:effect (req (let [watchId (keyword "respirocytes" (str (:cid (get-card state card))))]
```

Comment:
Just for consistency, we use `-` to separate words in identifier names, rather than camelCasing.  Also, you *probably* don't need to call `get-card` here... `get-card` works by finding a card in the state that matches the given card's `:cid`, so the returned map will have the same `:cid` as the `card` variable you already have access to.

Reply:
Awesome, thanks for the feedback.  Incorporated.